src/scripts/history.ts

```diff
@@ -2,15 +2,26 @@
 
 const HISTORY_LIMIT = 20;
 
+export interface HistoryEntry {
+    query: string;
+    result: string;
+}
+
+type HistoryUpdateCallback = (history: HistoryEntry[]) => void;
```

Comment:
Nit: Check the other instances as well.
```suggestion
type HistoryUpdateCallback = (history: Array<HistoryEntry>) => void;
```

Reply:
Used Array<> for this and other such instances.
https://github.com/dhruvdhaduk-simform/scientific-calc-ts/pull/1/commits/50c08cf264228375aa29f4d0adf10a03a46a20fe